system/nfsexport.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
```

Comment:
The shebang should always be #!/usr/bin/python, this allows ansible_python_interpreter to work[1]

[1] http://docs.ansible.com/ansible/developing_modules.html#module-checklist